core/api/src/graphql/public/types/object/consumer-account.ts

```diff
@@ -190,15 +203,10 @@ const ConsumerAccount = GT.Object<Account, GraphQLPublicContextAuth>({
           paginationArgs,
         })
 
-        if (error instanceof Error) {
+        if (!result || error) {
```

Comment:
if this use PartialResult then should use the new approach

Reply:
I will switch to using `type` on the partial result.